packages/babel-preset/src/types.ts

```diff
@@ -3,7 +3,11 @@ import type { EvalRule } from '@linaria/babel-preset';
 
 export type BabelPluginOptions = {
   /** Defines set of modules and imports handled by a transformPlugin. */
-  modules?: { moduleSource: string; importName: string }[];
+  modules?: {
+    moduleSource: string;
+    importName: string;
+    resetImportName?: string;
```

Comment:
```suggestion
    makeStylesimportName: string;
    makeResetStylesImportName?: string;
```

it's a bit longer, but it might make things clearer in the plugin code